geoconvert/convert.py

```diff
@@ -470,14 +496,14 @@ def address_to_country_code(text, lang=None):
 # - French postcode matching, because 5-digit postcodes are used in
 #   many countries.
 # - German postcode matching, because 5-digit postcodes are used in
-#   many countries and the postcodes do not follow landers boudaries
+#   many countries
 country_to_safe_subdivision_lookup_function = (
     ("BR", br_state_name_to_state_code),
     ("BR", br_postcode_to_state_code),
     ("CA", ca_postcode_to_province_code),
     ("CA", ca_province_name_to_province_code),
     ("FR", fr_dept_name_to_dept_code),
-    ("DE", de_address_to_land_code),
+    ("DE", de_safe_address_to_land_code),
```

Comment:
And here, instead of having that single safe function for Germany, you could have two entries, one for `de_land_name_to_land_code` and another for `de_hauptstadt_to_land_code`.